src/collectors/windows.plugin/perflib-storage.c

```diff
@@ -261,19 +255,18 @@ static inline void netdata_set_hd_usage(PERF_DATA_BLOCK *pDataBlock,
     // Description of incompatibilities present in both methods we are using
     // https://devblogs.microsoft.com/oldnewthing/20071101-00/?p=24613
     // We are using the variable that should not be affected by qyota ()
-    if ((GetDriveTypeA(path) != DRIVE_FIXED) || !GetDiskFreeSpaceExA(path,
+    if ((GetDriveTypeA(path) == DRIVE_UNKNOWN) || !GetDiskFreeSpaceExA(path,
```

Comment:
Can you briefly explain this change? Why? Doesn't look like a problem to me, but I'd like to hear "why".

Reply:
Sure. If the driver cannot be identified, it indicates that the next function could fail.

If the driver can be identified, I try to get the data directly from `GetDiskFreeSpaceExA`. If that fails, I use Perflib.

The idea is to use `Perflib` only if there is no way to get the data from the Windows API.